docs/faq.md

```diff
@@ -75,13 +75,27 @@ to misunderstand configuration details that can have an impact on the results.
 If you can, please reach out to the authors to confirm your configuration looks
 good to them.
 
-## I'd like to get my fuzzer evaluated on the free FuzzBench service, but I don't want the results to be public yet.
-
-Please reach out to us at fuzzbench@google.com.
-
-## I'm working on a new tool, but I'm not ready to make my code public yet. Can I use the FuzzBench service?
-
-Yes, please reach out to us at fuzzbench@google.com.
+## I'd like to get my fuzzer evaluated, but I don't want the results and/or code to be public yet. Can I use the FuzzBench service?
+
+Probably yes. We run private experiments for this purpose.
+Please reach out to us at fuzzbench@google.com. If we agree to benchmark your
+fuzzer, please follow the guide on
+[adding a new fuzzer]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/)
+on how to integrate your fuzzer with FuzzBench.
+
+You can ignore the sections on [Requesting an experiment]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/#requesting-an-experiment) and
+[Submitting your integration]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/#submitting-your-integration).
+Please test your fuzzer builds and works with our benchmarks, we won't have
+CI to do this for private experiments.
+Ideally, you should test all benchmarks using `make -j test-run-$FUZZER-all`.
+This takes too long on most machines so you should try a few of them:
```

Comment:
do some of the benchmarks tend to be more "problematic" then others? this is probably hard to tell but if you happen to have some in mind consider suggesting to prioritize testing with those.

Reply:
The ones I picked were probably some of the less problematic ones but are pretty fast. bloaty_fuzz_target might be more problematic but is a bit slower. Maybe I should point out that if your fuzzer doesn't change the build process with AFL/some other fuzzer we already have you probably don't need to test others but if it does change a lot (e.g. KLEE) you should test others like bloaty_fuzz_target?

Reply:
or just suggest of shortlist of benchmarks that includes a few fast ones and a few potentially problematic ones too